src/wallet/test/init_test_fixture.h

```diff
@@ -18,7 +18,6 @@ struct InitWalletDirTestingSetup: public BasicTestingSetup {
     fs::path m_datadir;
     fs::path m_cwd;
     std::map<std::string, fs::path> m_walletdir_path_cases;
-    NodeContext m_node;
```

Comment:
In commit "test: Avoid overwriting the NodeContext member of the testing setup" (fa00c37cba98794c1c12c4b77e885d831d9bf183)

Wow, good catches. I wonder if there was a compiler warning we could enable that would have caught these.